src/components/MapViewer/MenuWidget.jsx

```diff
@@ -2083,23 +2102,41 @@ class MenuWidget extends React.Component {
       .catch(() => {});
   };
   async fullExtent(elem) {
-    let parentId = elem.getAttribute('parentid');
+    this.findCheckedDataset(elem);
+    let BBoxes = {};
+    let firstLayer;
+    if (this.url.toLowerCase().includes('globaldynamiclandcover')) {
+      this.findDatasetBoundingBox(elem);
+      BBoxes = this.parseBBOXJSON(this.dataBBox);
+    } else if (this.url.toLowerCase().includes('globalland.vgt.vito')) {
```

Comment:
@Urkorue @manuelmartinezbilbomatica we should handle this kind of things with care, if the URL changes for whatever reason this implementation will not work.